src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,252 @@
+package Duke;
+
+import Duke.Exception.DataCorruptedException;
+import Duke.Exception.EmptyArgumentException;
+import Duke.Exception.WrongArgumentException;
+import Duke.Tasks.TaskType;
+import Duke.Tasks.TaskList;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.Scanner;
+
+
+public class Duke {
+    private static TaskList list = new TaskList();
+    private static boolean toSave = true;
+
+    private static final String SEPARATOR = "____________________________________________________________";
+
+    public static void main(String[] args) {
+        final String DUKE_LOGO =  " ____        _\n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        final String CONV_START = DUKE_LOGO + "Hello! I'm Duke";
+        final String CONV_END = "Bye. Hope to see you again soon!";
+
+
+        printOutput(CONV_START);
+
+        boolean initialiseSuccessful = true;
```

Comment:
Similar to above, you may want to consider renaming the boolean variable to something such as `isInitSuccess`, so as to adhere to the coding standard.